release/models/bgp/openconfig-bgp-common-multiprotocol.yang

```diff
@@ -24,7 +24,13 @@ submodule openconfig-bgp-common-multiprotocol {
     for multiple protocols in BGP. The groupings are common across
     multiple contexts.";
 
-  oc-ext:openconfig-version "8.0.0";
+  oc-ext:openconfig-version "9.0.0";
```

Comment:
Hi, can you set the version number to 8.1.0 since this is merely adding a leaf and is not a breaking change?

Reply:
Done!